public/user_data.html

```diff
@@ -1,262 +1,274 @@
 <!DOCTYPE html>
 <html>
- <head>
-   <meta charset="utf-8">
-   <meta name="viewport" content="width=device-width, initial-scale=1">
-   <title>ASB API Web Demo</title>
-   <div id='tokenDisplay'>
-     <textarea id='tokenContainer'></textarea>
-   </div>
- </head>
- <body>
-   
-  <button id='generateToken'>Get User Token</button> 
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Logged In</title>
+  <textarea id='tokenContainer'></textarea>
+</head>
+
+<body>
+  <button id='generateToken'>Get User Token</button>
   <button id='getAdminPrivileges'>Get Admin Privileges</button>
-  <div> 
-    <label for = "userFile">Upload file </label>
-    <input type = "file" id = "userFile" name = "userFile">
-    <button onClick = "store()" id = "userFilebutton">Submit</button>
+  <button onClick="checkAdmin()" id='checkAdmin'> Check Privileges </button>
```

Comment:
why do we need a button for this? Can't we just run the check script after the page loads and then enable/disable the button?